pydash/numerical.py

```diff
@@ -383,6 +384,30 @@ def std_deviation(array):
     return math.sqrt(variance(array))
 
 
+def subtract(minuend, subtrahend):
+    """Subtracts two numbers.
+
+    Args:
+        minuend (int/float): The first number in a subtraction.
+        subtrahend (int/float): The second number in a subtraction.
+
+    Returns:
+        int/float: Returns the difference.
+
+    Example:
+
+        >>> subtract(10, 4)
+        6
+        >>> subtract(-10, 4)
+        -14
+        >>> subtract(4, -10)
+        14
+
+    .. versionadded:: TODO
+    """
+    return minuend - subtrahend
```

Comment:
Take a look at Lodash's "createMathOperation" function which they call for add, subtract, multiply, and divide. Handles a few of the edge cases around inputs.